mm2src/coins/eth.rs

```diff
@@ -1390,7 +1390,7 @@ impl SwapOps for EthCoin {
     }
 
     fn is_supported_by_watchers(&self) -> bool {
-        false
+        std::env::var("USE_WATCHER_REWARD").is_ok()
```

Comment:
Can we use coins configuration instead of re-fetching the environment?

Reply:
I did this to enable watcher rewards in certain tests and disable in others temporarily. Soon I will implement watcher rewards as an option for orders and there won't be need for this anymore.